src/OpenTelemetry.Api/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,327 @@
+OpenTelemetry.Trace.SpanAttributeConstants
+OpenTelemetry.Trace.SpanAttributes
+OpenTelemetry.Trace.SpanAttributes.Add(string key, bool value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, bool[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, double value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, double[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, long value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, long[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, string value) -> void
+OpenTelemetry.Trace.SpanAttributes.Add(string key, string[] values) -> void
+OpenTelemetry.Trace.SpanAttributes.SpanAttributes() -> void
+OpenTelemetry.Trace.SpanAttributes.SpanAttributes(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>> attributes) -> void
+OpenTelemetry.Trace.SpanContext
+OpenTelemetry.Trace.SpanContext.Equals(OpenTelemetry.Trace.SpanContext other) -> bool
+OpenTelemetry.Trace.SpanContext.IsRemote.get -> bool
+OpenTelemetry.Trace.SpanContext.IsValid.get -> bool
+OpenTelemetry.Trace.SpanContext.SpanContext(in System.Diagnostics.ActivityContext activityContext) -> void
+OpenTelemetry.Trace.SpanContext.SpanContext(in System.Diagnostics.ActivityTraceId traceId, in System.Diagnostics.ActivitySpanId spanId, System.Diagnostics.ActivityTraceFlags traceFlags, bool isRemote = false, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> traceState = null) -> void
+OpenTelemetry.Trace.SpanContext.SpanId.get -> System.Diagnostics.ActivitySpanId
+OpenTelemetry.Trace.SpanContext.TraceFlags.get -> System.Diagnostics.ActivityTraceFlags
+OpenTelemetry.Trace.SpanContext.TraceId.get -> System.Diagnostics.ActivityTraceId
+OpenTelemetry.Trace.SpanContext.TraceState.get -> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>>
+OpenTelemetry.Trace.SpanHelper
+OpenTelemetry.Trace.SpanKind
+OpenTelemetry.Trace.SpanKind.Client = 3 -> OpenTelemetry.Trace.SpanKind
+OpenTelemetry.Trace.SpanKind.Consumer = 5 -> OpenTelemetry.Trace.SpanKind
+OpenTelemetry.Trace.SpanKind.Internal = 1 -> OpenTelemetry.Trace.SpanKind
+OpenTelemetry.Trace.SpanKind.Producer = 4 -> OpenTelemetry.Trace.SpanKind
+OpenTelemetry.Trace.SpanKind.Server = 2 -> OpenTelemetry.Trace.SpanKind
+OpenTelemetry.Trace.Status
+OpenTelemetry.Trace.Status.Description.get -> string
+OpenTelemetry.Trace.Status.Equals(OpenTelemetry.Trace.Status other) -> bool
+OpenTelemetry.Trace.Status.IsOk.get -> bool
+OpenTelemetry.Trace.Status.IsValid.get -> bool
+OpenTelemetry.Trace.Status.StatusCode.get -> OpenTelemetry.Trace.StatusCode
+OpenTelemetry.Trace.Status.WithDescription(string description) -> OpenTelemetry.Trace.Status
+OpenTelemetry.Trace.StatusCanonicalCode
+OpenTelemetry.Trace.StatusCanonicalCode.Aborted = 10 -> OpenTelemetry.Trace.StatusCanonicalCode
```

Comment:
haven't we moved away from the grpc?

Reply:
yes and no....
yes, right now, otel.status_code = Ok,Unset,Error
no, we still have to map the StatusCanonicalCode to Otel.Status_code. And, I think we still have to send to otlpexporter so it will work for old / new collectors

Reply:
They published new protos where we have:
status = otel.status_code
deprecated_status = grpc status

Reply:
should it be implementtion detail in OTLP exporter?

Reply:
yes, we can move this and the helper method that uses it.
Question: should i create a new pr with this changes pointing to master, then rebase this branch from master OR should i add all this changes here?
Why: if we keep doing things here, we will have issues when merging if takes too long.

Reply:
as I mentioned - I'm fine merging this PR as is. It was handy to comment here as all APIs are easily commenteable

Reply:
hello @SergeyKanzhelev , im working on separate mini-prs to solve this. already made some. and will keep updating here. :)

Reply:
fixed!